### PR TITLE
Feat : Added Modular CSS and Empty States

### PR DESCRIPTION
**Pull request by srejitk**
The feature request adds the following

1. Breaking down CSS to Modular CSS
2. Adding Empty state when no product in Cart/Wishlist/Listing
3. Updated Landing Page Components

### CODE REVIEW

**Review by nakulsharma15 — APPROVED** [review]
No changes suggested. Liked the overall folder structure and naming of variables.

File: House-of-glass/src/components/Cards/CartCard/CartCard.jsx [hunk 1]
```diff
@@ -1,68 +1,61 @@
...
+    useCart();
 
   return (
-    <div className="card card--horizontal box-shadow">
-      <div className="card__contents">
-        <div className="card__content--column card__text">
-          <h5 className="header-5">{name}</h5>
+    <div className=" card card--vertical box-shadow">
+      <div className="card__cover cover--vertical image--responsive">
+        <img src={imageUrl} />
```
Comment (coldpigli) [inline House-of-glass/src/components/Cards/CartCard/CartCard.jsx:14]: put an alt text

**Review by coldpigli — APPROVED** [review]
The CSS looks clean and modular. Good naming conventions

**Review by tanishq20 — APPROVED** [review]
Like the way you imported module CSS. Made your code more dynamic.

**Pull request merged**